reliontomo/protocols/protocol_refine_subtomograms.py

```diff
@@ -293,6 +293,9 @@ def _validate(self):
         if abs(inParticlesSRate - refVolumeSRate) >= sRateTol:
             errorMsg.append(f'The introduced particles and the reference volume must have the same sampling rate:\n'
                             f'{inParticlesSRate:.3f} != {refVolumeSRate:.3f} Å/px')
+        # Number of MPI must be at least 3
```

Comment:
this validation can be applied also to the 3d classification

Reply:
Classify3D protocol inherits from Refine Subtomograms protocol, so both cases are currently covered